india_compliance/patches/post_install/add_hsn_to_item_variant_copy_fields.py

```diff
@@ -0,0 +1,11 @@
+import frappe
+
+"""
+This patch makes sure hsn code is copied over to variant item when created from template
+"""
+
+
+def execute():
+    frappe.get_doc("Item Variant Settings").append(
+        "fields", {"field_name": "gst_hsn_code"}
+    ).save()
```

Comment:
Should be done only if it's not already there?

Reply:
Have added a check :)
thanks!